lib/Driver/WindowsToolChains.cpp

```diff
@@ -51,11 +51,7 @@ toolchains::Windows::addPluginArguments(const ArgList &Args,
   llvm::sys::path::remove_filename(LibraryPath); // Remove `swift`
 
   // In-process plugin server path.
-  SmallString<261> InProcPluginServerPath = LibraryPath;
-  llvm::sys::path::append(InProcPluginServerPath,
-                          "SwiftInProcPluginServer.dll");
-  Arguments.push_back("-in-process-plugin-server-path");
-  Arguments.push_back(Args.MakeArgString(InProcPluginServerPath));
+  addInProcPluginServerPath(LibraryPath, Args, Arguments);
```

Comment:
@rintaro what I was commenting in https://github.com/swiftlang/swift/pull/74740#discussion_r1656518148 about "not fully sure if the code around it is completely correct" is about the calculation of `PluginPath` and `LibraryPath`. The code right now is part of the `*ToolChains.cpp`, but I am not sure if it should move to a more central place as well, since I think it depends on the compiler host, not the target as well.

I can do the change, but I am not sure about its correctness, so I will wait until someone with more knowledge can say if this is the right place or it should be moved out of the `*ToolChains.cpp` and into the `ToolChains.cpp` file as well.